Dialogflow-webhook/finFraudwebhooks.js

```diff
@@ -175,230 +181,309 @@ export const createDocWithAssistant = async (req, res) => {
+                            fileURL = constants.PUBLIC_BUCKET_URL + '/' + threadId + '/' + threadId + constants.GPT3_5_FINE_TUNED + letterType + '.docx';
+
+                            break;
+                        case "Police Complaint":
+                            letterType = "PoliceComplaint";
+                            createLetterWithGPT3_5_Failed_txn(letterType, userInputData, threadId);
+                            textResponse = 'Creating Police Complaint, Please wait'
+                            docName = 'Police Complaint letter';
+                            fileURL = constants.PUBLIC_BUCKET_URL + '/' + threadId + '/' + threadId + constants.GPT3_5 + letterType + '.docx';
+                            break;
+                        case "Consumer court":
+                            letterType = "ConsumerCourt";
+
+                            createConsumerCourtLetterwithFineTuned(letterType, userInputData, threadId);
+                            textResponse = 'Creating Consumer Court letter, Please wait'
+                            docName = 'Consumer Court letter';
+                            fileURL = constants.PUBLIC_BUCKET_URL + '/' + threadId + '/' + threadId + constants.GPT3_5_FINE_TUNED + letterType + '.docx';
+
+                            break;
+                        case "RTI Application":
+                            if (["Bank of Baroda",
+                                    "Bank of India",
+                                    "Bank of Maharashtra",
+                                    "Canara Bank",
+                                    "Central Bank of India",
+                                    "Indian Bank",
+                                    "Indian Overseas Bank",
+                                    "Punjab and Sind Bank",
+                                    "Punjab National Bank",
+                                    "State Bank of India",
+                                    "UCO Bank",
+                                    "Union Bank of India"
+                                ].includes(sessionInfo.parameters.name_of_bank)) {
+                                letterType = 'RTI'
+                                createLetterWithGPT3_5_Failed_txn(letterType, userInputData, threadId)
+                                textResponse = 'Creating RTI Application, Please wait'
+                                docName = 'RTI letter';
+                                fileURL = constants.PUBLIC_BUCKET_URL + '/' + threadId + '/' + threadId + constants.GPT3_5 + letterType + '.docx';
+
+
+                            } else {
+                                textResponse = 'RTI Application is applicable only for Public Sector Banks';
+                                let targetPage = "projects/atmprebuiltagent/locations/asia-south1/agents/9d9f910c-d14b-4489-b1f9-98c6c3e67c61/flows/aa979626-23c2-4a9c-b456-45403d2dbf61/pages/43328113-8eb4-4cd5-911a-40700ad78d5c";
```

Comment:
targetPage ID for failed transaction will be different
` projects/atmprebuiltagent/locations/us-central1/agents/7f79f692-4e94-4d04-9bf1-203e68b815dc/flows/10d45ea1-8174-4894-96ed-0bbfb9b802ae/page/43328113-8eb4-4cd5-911a-40700ad78d5c`